packages/embark-swarm/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+# [4.1.0-beta.3](https://github.com/embark-framework/embark/compare/v4.1.0-beta.2...v4.1.0-beta.3) (2019-06-07)
+
+**Note:** Version bump only for package embark-swarm
```

Comment:
Same here, this file will be created by lerna when we do another release. I guess we can remove it from this PR. No biggie if we keep it though.